site/src/Tour.jsx

```diff
@@ -35,6 +36,24 @@ const Steps = [
     placement: "auto",
     offset: 0,
   },
+  {
+    target: ".tour-layers-checkboxes",
+    content:
+      "Altering the visibility at the theme level is available, as well as at a more granular type level",
+    disableBeacon: true,
+    title: "Change Layer Visibility",
+    placement: "auto",
+    offset: 0,
+  },
+  {
+    target: ".tour-layers-pins",
+    content:
+      "This button sets a theme as active. Active themes are highlighted on the map and prioritized over non-active themes for click events.",
```

Comment:
Replace 'click events', as this phrase is too developer-y (an easy trap to fall into, I know!) 

`"This button sets a theme as active. Active themes are highlighted and receive click-to-select priority",`